src/views/Provisioners/ViewWorker/index.jsx

```diff
@@ -0,0 +1,59 @@
+import { PureComponent, Fragment } from 'react';
+import { hot } from 'react-hot-loader';
+import { graphql } from 'react-apollo/index';
+import { withStyles } from 'material-ui/styles';
+import Dashboard from '../../../components/Dashboard';
+import Spinner from '../../../components/Spinner';
+import ErrorPanel from '../../../components/ErrorPanel';
+import WorkerDetailsCard from '../../../components/WorkerDetailsCard';
+import WorkerTable from '../../../components/WorkerTable';
+import workerQuery from './worker.graphql';
+
+@hot(module)
+@graphql(workerQuery, {
+  skip: ({ match: { params } }) => !params.provisionerId,
+  options: ({ match: { params } }) => ({
+    variables: {
```

Comment:
Would this work by passing `params` as `variables`?

```js
variables: params,
```

Reply:
It does!